test/index.ts

```diff
@@ -18,11 +21,56 @@ describe("Hello-world-teste", () => {
 });
 
 describe("Hello-world-graphql", () => {
-  before(async () => {
-    await startServer();
-  });
+  const graphqlQuery = {
+    query: `query fetchHelloWord {
+    hello
+  }`,
+  };
   it("Should return Hello world!", async () => {
     const response = await axios.post(endpoint, graphqlQuery);
     expect(response.data.data.hello).to.be.eq("Hello world!");
   });
 });
+
+describe("CreateUser-mutation-test", () => {
+  const user: UserInput = {
+    birthDate: "07-12-2003",
+    email: "matheus.12345@hotmail.com",
+    name: "Matheus",
+    password: "Teste1235571",
+  };
+  const graphqlQueryMutation = {
+    query: `mutation Mutation($data: UserInput!) {
+      createUser(data: $data) {
+        birthDate
+        email
+        id
+        name
+        password
+      }
+    }`,
+    variables: {
+      data: user,
+    },
+  };
+  it("Should create a user in data base and return that info", async () => {
+    const response = await axios.post(endpoint, graphqlQueryMutation);
+    expect(response.data.data.createUser.birthDate).to.be.eq(user.birthDate);
+    expect(response.data.data.createUser.email).to.be.eq(user.email);
+    expect(response.data.data.createUser.name).to.be.eq(user.name);
+    expect(
+      await compare(user.password, response.data.data.createUser.password),
+    ).to.be.eq(true);
+  });
+  after(async () => {
```

Comment:
```suggestion
  afterEach(async () => {
```
pensando que depois teremos mais its (um pra cada cenário dessa mutation)
e acho que pode deixar la em cima mesmo